laboratory-service/src/main/java/com/raf/si/laboratoryservice/dto/response/DepartmentResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.raf.si.laboratoryservice.dto.response;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode
+public class DepartmentResponse {
+    private UUID pbo;
+    private String name;
+}
```

Comment:
Department response je promenjen na user-service, ima dodatno polje bolnica kome pripada odeljenje, u slucaju da ti taj property ne treba, okej je da se ide sa ovim?

Reply:
Koliko znam mislim da nam nije potrebno to polje tako da mislim da je ovo okej